Assets/VIRTUOSO/Prefabs/[UnityBase_CameraRig].prefab

```diff
@@ -285,11 +285,11 @@ Camera:
   m_TargetEye: 3
   m_HDR: 0
   m_AllowMSAA: 1
+  m_AllowDynamicResolution: 0
   m_ForceIntoRT: 0
   m_OcclusionCulling: 1
   m_StereoConvergence: 10
   m_StereoSeparation: 0.022
-  m_StereoMirrorMode: 0
 --- !u!54 &54108975209875660
```

Comment:
When I imported this prefab, the 'HandNodeType' variable on the SDK_UnityControllerTracker script were not set to the proper hands and set to the default 'LeftEye'.